torch/csrc/jit/tensorexpr/operators/quantization.cpp

```diff
@@ -183,6 +209,47 @@ Tensor computeQuantizePerTensor(
       c10::nullopt,
       qscale.node(),
       qzero.node());
+  return Tensor(buf, vars, e.node());
+}
+
+Tensor computeQuantizedAdd(
+    const std::vector<ArgValue>& inputs,
+    const std::vector<ExprHandle>& outputShape,
+    const c10::optional<ScalarType>& outputType,
+    at::Device) {
+  const BufHandle& QA = c10::get<BufHandle>(inputs[0]);
+  const BufHandle& QB = c10::get<BufHandle>(inputs[1]);
+  auto qa_scale = ExprHandle(QA.node()->qscale());
+  auto qa_zero = ExprHandle(QA.node()->qzero());
+  auto qb_scale = ExprHandle(QB.node()->qscale());
+  auto qb_zero = ExprHandle(QB.node()->qzero());
+  ExprHandle out_qscale = DoubleImm::make(c10::get<double>(inputs[2]));
+  ExprHandle out_qzero = LongImm::make(c10::get<int64_t>(inputs[3]));
+  Dtype dequant_dtype = kFloat;
+  Dtype out_dtype = outputType ? Dtype(*outputType) : QA.dtype();
+  std::vector<VarPtr> vars;
+  std::vector<ExprHandle> indices;
+  for (const auto& os : outputShape) {
+    auto var = alloc<Var>("", os.node()->dtype());
+    vars.push_back(var);
+    indices.push_back(VarHandle(var));
+  }
+  auto lhs = tensorOrConstant(inputs[0], indices);
+  auto rhs = tensorOrConstant(inputs[1], indices);
+  ExprHandle exprHandle = quant(
+      dequant(lhs, dequant_dtype, qa_scale, qa_zero) +
+          dequant(rhs, dequant_dtype, qb_scale, qb_zero),
+      out_dtype,
+      out_qscale,
+      out_qzero);
+  BufPtr buf = alloc<Buf>(
+      "quantized_add",
+      ExprHandleVectorToExprVector(outputShape),
+      out_dtype,
+      nullptr,
+      c10::nullopt,
+      out_qscale.node(),
+      out_qzero.node());
```

Comment:
I suspect it would be more compact if we used `Compute` API to create the result `Tensor`.